apps/omg_watcher_rpc/lib/application.ex

```diff
@@ -50,6 +50,8 @@ defmodule OMG.WatcherRPC.Application do
       name: OMG.WatcherRPC.RootSupervisor
     ]
 
+    {_, _} = Logger.add_backend(Sentry.LoggerBackend)
```

Comment:
Does it make sense to match `{:ok, _}` on these? If not, should we handle errors?

Reply:
good point, definitely matching on `:ok` makes sense!  handling errors not so, if it doesn't work there's not much we can do and we want to fail completely.

Reply:
Yep agreed. The reason why I went about this route is I wasn't entirely sure where the best place to put this code so and did a general match instead so I could place it everywhere and not have it complain when I loaded it more than once. 🤕  If you all have suggestions where I should load this that would be great, otherwise I'll try to create another `application.ex` for the umbrella app. re: https://github.com/getsentry/sentry-elixir/issues/345.